### PR TITLE
Fenwick_Tree.php Added

### LINKED ISSUE #2173: Fenwick Tree
- [x] C
- [x] C++
- [ ] C# - @ritish099
- [ ] CoffeeScript
- [x] Dart - @Rukmini-Meda
- [ ] Go
- [x] Java
- [x] JS
- [x] Kotlin - @JasbirCodeSpace
- [x] Php - @goyalmuskan
- [x] Python
- [ ] Readme - @AJAYSHARMA2012
- [ ] Ruby
- [ ] Rust
- [ ] TypeScript

### PR DESCRIPTION
**Pull request by goyalmuskan**
Fixes #2173 

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [x] Input is taken dynamically.
- [x] Sample Input / Output is added at the end of file.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

#### Changes proposed in this pull request:
- Fenwick_Tree.php Added

#### Languages Used:
- PHP

#### Files Added:
- Fenwick_Tree.php

> We're happy to help you get this ready -- don't be afraid to ask for help.

Thanks!

### CODE REVIEW

File: Fenwick_Tree/Fenwick_Tree.php [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+    }
+    // Return Fenwick Tree array
+    return $ft;
+}
+
+//Driver code to test above methods 
+function main() {
+    $array = array(2, 1, 1, 3, 2, 3, 4, 5, 6, 7, 8, 9); 
+    $n = construct($freq, sizeof($array)); 
+    echo implode(sum($n,5));
```
Comment (asha15) [inline Fenwick_Tree/Fenwick_Tree.php:86]: keep a space after comma

File: Fenwick_Tree/Fenwick_Tree.php [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+    $array[3] = $array[3] + 6;
+    update($n, sizeof($array), 3, 6) ;
+    echo implode(sum($n,5));
```
Comment (asha15) [inline Fenwick_Tree/Fenwick_Tree.php:89]: keep a space after comma

File: Fenwick_Tree/Fenwick_Tree.php [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+    from the given array of length "size"
+
+    Return : Fenwick Tree array ft[]
+    */    
+    // Init ft array of length 1000 with zero value initially 
+    $ft = array(); 
+    // Constructing Fenwick Tree by Update operation  
+    for( $i = 0; $i < $size; $i++) {
+      // Update operation
+      update($ft, $size, $i, $array[$i]);
```
Comment (asha15) [inline Fenwick_Tree/Fenwick_Tree.php:76]: please correct the indentation

File: Fenwick_Tree/Fenwick_Tree.php [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+        $index += $index and (-$index);
+    }
+  } 
+  
+  
+// Build function
+function construct($array, $size) { 
+    /*
+    Argument
+    arr : The original array
```
Comment (asha15) [inline Fenwick_Tree/Fenwick_Tree.php:64]: arr?

**Comment by goyalmuskan** [comment]
@asha15 I have made all the changes. :)

**Comment by asha15** [comment]
Good. please squash your commits and let me know

**Comment by goyalmuskan** [comment]
@asha15 Hey, I have squashed the commits! :)

**Pull request merged**